src/Uno.UI/UI/Xaml/Controls/TextBox/InputScopeHelper.Android.cs

```diff
@@ -21,27 +21,14 @@ internal static InputTypes ConvertToCapitalization(InputTypes types, InputScope
 			}
 		}
 
-		internal static InputTypes ConvertToRemoveSuggestions(InputTypes value) 
-			=> value | InputTypes.TextFlagNoSuggestions;
-
-		internal static InputTypes ConvertInputScope(InputScope value, bool isPassword = false)
+		internal static InputTypes ConvertToRemoveSuggestions(InputTypes value, bool forceRemove)
 		{
-			if (isPassword)
```

Comment:
Not validating for passwords anymore ?

Reply:
I removed an overload that wasn't used anywhere, that coincidentally had the same signature as the one I added